cms/djangoapps/contentstore/views/tests/test_course_index.py

```diff
@@ -233,7 +233,7 @@ def test_notifications_handler_dismiss(self):
             # delete nofications that are dismissed
             CourseRerunState.objects.get(id=rerun_state.id)
 
-        self.assertFalse(has_course_author_access(user2, rerun_course_key))
+        self.assertTrue(has_course_author_access(user2, rerun_course_key))
```

Comment:
You could try contacting the author of this line of code-- DeRosier. It looks like it was very intentionally added.

Here is the PR: https://github.com/edx/edx-platform/pull/6123